src/app/modules/ontology-exploration/ontology-search/ontology-search.component.ts

```diff
@@ -9,23 +9,30 @@ import { OntologyNode } from '../../../core/models/ontology-node';
 import { OntologySearchService, SearchResult } from '../../../core/services/ontology-search/ontology-search.service';
 
 
+/**
+ * Componenet for searching the Ontology nodes.
+ */
 @Component({
   selector: 'ccf-ontology-search',
   templateUrl: './ontology-search.component.html',
   styleUrls: ['./ontology-search.component.scss']
 })
 export class OntologySearchComponent implements OnInit {
```

Comment:
Missing branch coverage for `displayFormatter` (line 73). I.e.  when `option === undefined` or `option.displayLabel === undefined`